src/main.cpp

```diff
@@ -399,15 +399,38 @@ int main(int argc, char* argv[])
     const bool firstInstanceOfMudlet = instanceCoordinator->tryToStart();
 
     const QStringList positionalArguments = parser.positionalArguments();
+    QString telnetUri;
+    
     if (!positionalArguments.isEmpty()) {
-        const QString absPath = QDir(positionalArguments.first()).absolutePath();
-        instanceCoordinator->queuePackage(absPath);
-        if (!firstInstanceOfMudlet) {
-            const bool successful = instanceCoordinator->installPackagesRemotely();
-            if (successful) {
-                return 0;
+        const QString firstArg = positionalArguments.first();
+        
+        // Check if it's a telnet:// URI
+        if (firstArg.startsWith("telnet://", Qt::CaseInsensitive)) {
```

Comment:
Don't let `QString`s be initialised by C raw-string literals - as other coders cannot always tell whether they are to be used internally {wrap them in `qsl(`...`)`} or need to be made available for **I18n**/**L10n** so need to be passed to those doing the translations for languages other than "English (American)" {wrap them in `tr(`...`)`}:
```suggestion
        if (firstArg.startsWith(qsl("telnet://"), Qt::CaseInsensitive)) {
```